pkg/configs/configs.go

```diff
@@ -27,9 +27,9 @@ type Config struct {
 // _version_ of a configuration a unique ID and guarantees that later versions
 // have greater IDs.
 type View struct {
-	ID        ID        `json:"id"`
-	Config    Config    `json:"config"`
-	DeletedAt time.Time `json:"deleted_at"`
+	ID        ID          `json:"id"`
+	Config    Config      `json:"config"`
+	DeletedAt pq.NullTime `json:"deleted_at"`
```

Comment:
I don't understand this.  Why can't the zero time stand for "null" ?

Reply:
We cannot store `null` time into `time.Time` variable.
Added description of the PR.

Reply:
The zero value is `time.Time{}`, and can be checked like `DeletedAt.IsZero()`

Reply:
Yes, zero time can stand for null, but opposite doesn't work: null cannot stand for zero time when I scan values from DB: 
```panic: sql: Scan error on column index 2: unsupported Scan, storing driver.Value type <nil> into type *time.Time```

Reply:
Fine; use a DB-related type in the DB code but don't export it into the rest of the code.

Reply:
We can use another variable to scan but I think it will complicate code.

Reply:
Putting an unfamiliar type in this struct is a huge overhead to understanding. Adding an extra variable in the scan code, which already has several variables for scanning only, is easy to understand.

Reply:
Fixed, PTAL